test/stats/statcmp.js

```diff
@@ -1,20 +1,12 @@
-/* eslint-disable no-var */
-
 "use strict";
 
-var fs = require("fs");
-
-try {
-  var ttest = require("ttest");
-} catch (e) {
-  console.log('\nttest is not installed -- to intall, run "npm install ttest"');
-  console.log("Continuing without significance test...\n");
-}
+const fs = require("fs");
+const ttest = require("ttest");
```

Comment:
This change is fine, since the `try...catch` construction pre-dates the introduction of proper package management by *years*; note that `ttest` is listed here:
https://github.com/mozilla/pdf.js/blob/c86f44f4233b892de6c5ea2882ca88d0e7d6d003/package.json#L54